src/Core/Validation/FieldValidation/EmailFieldValidator.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace SilverStripe\Core\Validation\FieldValidation;
+
+use Symfony\Component\Validator\Constraints;
```

Comment:
```suggestion
use Symfony\Component\Validator\Constraints\Email;
```
Other than in tests (where the intention is to avoid a bunch of use statements for test classes in the same namespaces), I can't think of anywhere we have a partial use statement like that.
If the intention is to avoid ambiguity, we tend to use aliases for that (`use something as somethingSpecific`)

Reply:
Has been updated